docs-api/frontity-plugins/embedded-mode.md

```diff
@@ -0,0 +1,69 @@
+# Frontity Embedded Mode
+
+This plugin enables an **Embedded Mode** implementation of Frontity. It can be downloaded from [here](https://github.com/frontity/frontity-embedded).
```

Comment:
@santosguillamot are we finally going to publish it in the WordPress repository?

If not, maybe we could add two links, one to the repository (code) and the other to the zip download, like this: 

```suggestion
This plugin enables an **Embedded Mode** implementation of Frontity. 

- [Explore the code](https://github.com/frontity/frontity-embedded)
- [Download the zip](https://github.com/frontity/frontity-embedded/archive/refs/heads/master.zip) 
```

Reply:
I like the idea of the direct link to the zip file. I didn't think of that possibility. 🤷🏻‍♂️

Reply:
Though if it's going to be submitted to the plugin repository that will be even better. However that will also depend on it being accepted by the moderators.

Reply:
>@SantosGuillamot are we finally going to publish it in the WordPress repository?

Yes, my idea is to publish it in the repository once all [the issues](https://github.com/frontity/frontity-embedded) are merged. I've made a Pull Request for the last one by the way.

Reply:
Approved 👍

Reply:
Done: https://github.com/frontity/api-reference/pull/34/commits/7cb15b98b19814873209110917d5fedfa98a1801